File: leetCode/fukun/301-400/lc_344.js
var reverseString = function (s) {
    for (let i = 0; i < s.length / 2; i++) {
        let tmp = s[i];
        s[i] = s[s.length - 1 - i];
        s[s.length - 1 - i] = tmp;
    }
    return s;
};
console.log(reverseString(["h", "e", "l", "l", "o"]));

File: leetCode/lwc/0301-0400/lc_344.java
package leetcode;

/**
 * @author lwc
 * @date 2019/12/9 17:19
 */
public class lc_344 {
    public void reverseString(char[] s) {
        int len = s.length;
        for(int i = 0; i < len / 2; i++){
            char temp = s[i];
            s[i] = s[len - 1 - i];
            s[len - 1 - i] = temp;
        }
    }
}
